Jenkinsfile

```diff
@@ -64,7 +65,7 @@ webappPipeline {
         return readJSON(file: 'package.json').version
     }
     team = 'Core UI'
-    mailer = 'matthew.cheely@genesys.com, daragh.king@genesys.com, jordan.stith@genesys.com, thomas.dillon@genesys.com, katie.bobbe@genesys.com, gavin.everett@genesys.com, jason.evans@genesys.com'
+    mailer = 'CoreUI@genesys.com'
```

Comment:
This was a workaround for a bug (emails to a mailing list not working) so we will need to check if it is working now.

Reply:
I was the one who fixed that issue and found out it was because jenkins emails were being blocked due to settings. It's already been tested with Nav-System that it works now